Tests/UpstreamIntegrationTests/util/JSONUtils.swift

```diff
@@ -0,0 +1,162 @@
+//
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+
+import Foundation
+import XCTest
+
+enum JSON: Equatable {
```

Comment:
This class seems to provide similar support as [AnyCodable](https://github.com/adobe/aepsdk-core-ios/blob/main/AEPServices/Sources/utility/AnyCodable.swift) from Core, can we reuse that and if needed add just the necessary wrappers over the actual and expected values to be able to compare them using the implicit equality operator from AnyCodable?

Reply:
For decoding inner arrays as [AnyCodable] seems worth considering adding that functionality in AEPServices